src/views/catalog/wizard/tabs/network/utils/utils.ts

```diff
@@ -0,0 +1,18 @@
+import produce from 'immer';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { getInterface } from '@kubevirt-utils/components/NetworkInterfaceModal/utils/helpers';
+import { NetworkInterfaceState } from '@kubevirt-utils/components/NetworkInterfaceModal/utils/types';
+import { getInterfaces } from '@kubevirt-utils/resources/vm';
+import { ensurePath } from '@kubevirt-utils/utils/utils';
+
+export const setInterfaceLinkState = (
+  vm: V1VirtualMachine,
+  nicName: string,
+  desiredState: NetworkInterfaceState,
+) =>
+  produce(vm, (draftVM) => {
+    ensurePath(draftVM, ['spec.template.spec.domain.devices.interfaces']);
+    const interfaces = getInterfaces(draftVM);
+    getInterface(interfaces, nicName).state = desiredState;
```

Comment:
can u separate the two lines? I think it is more readable

Reply:
I don't understand what you're asking for. Do you mean add an empty line after the `ensurePath` call?